docs/UserGuide.md

```diff
@@ -79,16 +79,49 @@ Adds a client to the address book.
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <box type="tip" seamless>
+    Parameters
+</box>
+
+1. `INDEX` : Must be a valid client index in the filtered list that is currently shown.
+2. `NAME` : Must be alphanumeric, can include spaces.
+3. `PHONE_NUMBER` : Must be numeric and at least 3 digits long.
+4. `EMAIl` : Must ba a valid email address following the format `local-part@domain`.
+5. `ADDRESS`: Can take any values.
```

Comment:
maybe we can exclude characters like "/" though for address.

Reply:
Addresses can take "/"